core/node/state_keeper/src/mempool_actor.rs

```diff
@@ -122,16 +123,22 @@ impl MempoolFetcher {
                 )
                 .await
                 .context("failed syncing mempool")?;
+
+            let transactions = transactions_with_constraints
+                .iter()
+                .map(|(t, _c)| t)
+                .collect_vec();
```

Comment:
Bikeshedding: It's easy to avoid dependency on `Itertools` by using `collect` with a type annotation or a turbofish.

Reply:
changed